hack/checkout_account.py

```diff
@@ -27,24 +27,29 @@
 RESOURCE_TYPE = "aws-account"
 USER = "cluster-api-provider-aws"
 
-if __name__ == "__main__":
-    conn = httplib.HTTPConnection(BOSKOS_HOST)
-
+def post_request(input_state = "free"):
     conn.request("POST", "/acquire?%s" % urllib.urlencode({
         'type': RESOURCE_TYPE,
         'owner': USER,
-        'state': 'free',
+        'state': input_state,
         'dest': 'busy',
     })
 
     )
-    resp = conn.getresponse()
+    return conn.getresponse()
+
+if __name__ == "__main__":
+    conn = httplib.HTTPConnection(BOSKOS_HOST)
+
+    resp = post_request()
+    if resp.status == 404:
+        resp = post_request("clean")
+
     if resp.status != 200:
         sys.exit("Got invalid response %d: %s" % (resp.status, resp.reason))
 
     body = json.load(resp)
     conn.close()
     print 'export BOSKOS_RESOURCE_NAME="%s";' % body['name']
     print 'export AWS_ACCESS_KEY_ID="%s";' % body['userdata']['access-key-id']
-    print 'export AWS_SECRET_ACCESS_KEY="%s";' % body['userdata']['secret-access-key']
-
+    print 'export AWS_SECRET_ACCESS_KEY="%s";' % body['userdata']['secret-access-key']
```

Comment:
Does your editor remove newlines at the end of the file?

Reply:
Not sure. Will add back.